pkg/autokitteh/builtins/@http.py

```diff
@@ -1,8 +1,52 @@
-def delete(url, params, headers, body, form_body, form_encoding, json_body, auth):
-    """HTTP Delete method request
-    API:
-      https://datatracker.ietf.org/doc/html/rfc9110#name-delete"""
-    pass
+def get(url, params, headers, body, form_body, form_encoding, json_body, auth):
+    """HTTP Get method request
+
+   Args:
```

Comment:
I'm not sure whether this will work. LSP parses doc section, and not showing all subsections. This requires re-checking in LSP

Reply:
I rechecked this in LSP and it seems that Args and Returns section will be shown, while Examples will be excluded in hover.

Reply:
pay attention that there are 2 different completion scenarios. Symbol and hover.
- for example for get method. symbols will be triggered on `get(` then you'll get text version of doc part
- hover will be triggered by placing the mouse on the symbol. Then you'll get the markdown version of some other part

Reply:
I will open PR soon (maybe tomorrow) that will fix passing more blocks to hover. Not only `args` and `returns`